### PR TITLE
(tlg0561)(tlg0641)(tlg0655)

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated cts_.xml and several files in Erotici Scriptores Graeci, Volume 1 and added in Greek titles.

Also wondering why updated ctsized file tlg0641.tlg001.perseus-grc2 is not showing up in Scaife.

### CODE REVIEW

**Comment by lcerrato** [comment]
@AlisonBabeu 
I see tlg0641 in Scaife. Maybe a loading issue?

**Review by lcerrato — CHANGES_REQUESTED** [review]

File: data/tlg0561/tlg001/tlg0561.tlg001.perseus-grc2.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
- <teiHeader>
+ <teiHeader xml:lang="eng">
  <fileDesc>
 <titleStmt>
-<title>Daphnis and Chloe</title>
+ <title xml:lang="lat">Δάφνις καὶ Χλόη</title>
```
[the remark this answers is not in the document]
Comment (AlisonBabeu) [reply data/tlg0561/tlg001/tlg0561.tlg001.perseus-grc2.xml:7]: Duh! Thanks!

**Review by lcerrato — APPROVED** [review]

**Comment by lcerrato** [comment]
waiting on hook test

File: data/tlg0561/tlg001/tlg0561.tlg001.perseus-grc2.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (AlisonBabeu) [reply data/tlg0561/tlg001/tlg0561.tlg001.perseus-grc2.xml]: @lcerrato its tlg0655.tlg001 that isn't showing up! Double duh!

**Comment by lcerrato** [comment]
@AlisonBabeu tlg0655.tlg001 is not in the release so there is something non-compliant about it

**Comment by lcerrato** [comment]
@AlisonBabeu There may be something in the structure of tlg0655.tlg001 but I do not see any obvious issues.

**Comment by lcerrato** [comment]
opened https://github.com/PerseusDL/canonical-greekLit/issues/1449 for investigation post merge

**Pull request merged**